hr_employee_calendar_planning/views/hr_employee_views.xml

```diff
@@ -14,7 +14,10 @@
                     <tree editable="top">
                         <field name="date_start" />
                         <field name="date_end" />
-                        <field name="calendar_id" />
+                        <field
+                            name="calendar_id"
+                            domain="['|', ('company_id', '=', False), ('company_id', '=', parent.company_id)]"
```

Comment:
Why not doing it at field level? And what about `check_company=True`?

Reply:
If we add check_company=True to `calendar_id` field we will need to add `company_id` field (and add these field in view as invisible) and set as related (employee_id.company_id) but don't work because is not set before record creation.

Reply:
No problem on that, as AFAIK, the constraint is checked after that assignation. I think we should stick maximum to the standard multi-companies mechanisms.

Reply:
Ok, but if the `company_id` field is not set, no calendar will be show (because all records have some company defined).